### ISSUE TITLE
yarn install fails

### ISSUE THREAD

**Issue by freddi301** [radicle-dev/radicle-upstream#2554]
➤ YN0066: │ typescript@patch:typescript@npm%3A4.4.3#~builtin<compat/typescript>::version=4.4.3&hash=d8b4e7: Cannot apply hunk #14
➤ YN0018: │ radicle-avatar@https://github.com/radicle-dev/radicle-avatar.git#commit=28033ef5a562aeb52c2e77c008021d27c3b24f4e: The remote archive doesn't match the expected checksum
➤ YN0013: │ radicle-contracts@https://github.com/radicle-dev/radicle-contracts.git#commit=157a5b59df94704702623765198deb4ba70ace84 can't be found in the cache  
➤ YN0018: │ radicle-contracts@https://github.com/radicle-dev/radicle-contracts.git#commit=157a5b59df94704702623765198deb4ba70ace84: The remote archive doesn't match the expected checksum

**Comment by rudolfs** [comment]
Have you `yarn install`ed from the repo before? It might be because we force-pushed some commits and there's a mismatch between your yarn cache and what's available in the repo. You could try to run ` yarn cache clean`.

**Comment by freddi301** [comment]
It was a fresh install, anyway retried anew with cache clean

➤ YN0013: │ radicle-avatar@https://github.com/radicle-dev/radicle-avatar.git#commit=28033ef5a562aeb52c2e77c008021d27c
➤ YN0018: │ radicle-avatar@https://github.com/radicle-dev/radicle-avatar.git#commit=28033ef5a562aeb52c2e77c008021d27c3b24f4e: The remote archive doesn't match the expected checksum
➤ YN0013: │ radicle-contracts@https://github.com/radicle-dev/radicle-contracts.git#commit=157a5b59df94704702623765198
➤ YN0018: │ radicle-contracts@https://github.com/radicle-dev/radicle-contracts.git#commit=157a5b59df94704702623765198deb4ba70ace84: The remote archive doesn't match the expected checksum

**Comment by rudolfs** [comment]
Could you try one more thing?
```
yarn cache clean --all
yarn install
```

I assume you're trying to build from latest `master`?

**Comment by freddi301** [comment]
Yes, building master.
tried but still similar errors

➤ YN0066: │ typescript@patch:typescript@npm%3A4.4.4#~builtin<compat/typescript>::version=4.4.4&hash=d8b4e7: Cannot apply hunk #14
➤ YN0013: │ radicle-avatar@https://github.com/radicle-dev/radicle-avatar.git#commit=28033ef5a562aeb52c2e77c008021d27c3b24f4e can't be found in the cache and will be fetched from the remote reposit 
➤ YN0018: │ radicle-avatar@https://github.com/radicle-dev/radicle-avatar.git#commit=28033ef5a562aeb52c2e77c008021d27c3b24f4e: The remote archive doesn't match the expected checksum
➤ YN0013: │ radicle-contracts@https://github.com/radicle-dev/radicle-contracts.git#commit=157a5b59df94704702623765198deb4ba70ace84 can't be found in the cache and will be fetched from the remote r 
➤ YN0018: │ radicle-contracts@https://github.com/radicle-dev/radicle-contracts.git#commit=157a5b59df94704702623765198deb4ba70ace84: The remote archive doesn't match the expected checksum

**Comment by freddi301** [comment]
yarn 3.0.1
windows 10

**Comment by rudolfs** [comment]
I'm running yarn `3.1.0` locally (macOS).
Might try to upgrade. 🤷

**Comment by rudolfs** [comment]
Another idea would be to try out this branch, once it's out of draft and passes CI: https://github.com/radicle-dev/radicle-upstream/pull/2574. We upgraded the package manager there.

**Comment by freddi301** [comment]
> 
> 
> I'm running yarn `3.1.0` locally (macOS). Might try to upgrade. 🤷

Still same errors

**Comment by rudolfs** [comment]
Thanks for trying things out!
I'm out of ideas of what the cause could be. 😞

@geigerzaehler do you maybe have an idea?

**Comment by freddi301** [comment]
Thank you for support! is there any collaborator that succesfully installed on windows?
If yes then i assume it's local issue, thank you.
Amazing work , keep going 😄

**Comment by geigerzaehler** [comment]
I have a couple of ideas but they’re mostly stabs in the dark
* Run `env` to see if there’s an environment variable that enables a specific yarn setting
* Post the output of `yarn config`
* Delete `~/.yarn/berry/cache` and `~/.cache/yarn`.
* Try running it in a docker image

**Comment by rudolfs** [comment]
Yeah, @bltavares originally implemented the experimental Windows support: https://github.com/radicle-dev/radicle-upstream/pull/2375#issuecomment-927937775.

That was a while ago though. Might be that we broke something along the way and didn't notice, since we don't yet have a Windows CI build or anybody actively using it on Windows for development.

**Comment by rudolfs** [comment]
@freddi301 https://github.com/radicle-dev/radicle-upstream/pull/2574 got merged, could you try building the latest master now?

**Comment by freddi301** [comment]
Still same errors :(
Feel free to close issue till windows CI is set up.

**Comment by rudolfs** [comment]
Ok. Thanks for trying out anyway!
I'll leave it open, in case anyone else on Windows bumps into this.

**Comment by jakoritarleite** [comment]
When using yarn on Windows, there's a bug that you need to define an environment variable to ignore the checksum. See [#2375](https://github.com/radicle-dev/radicle-upstream/pull/2375#issuecomment-927937775) for more information.
